### ISSUE TITLE
Namespace for array element - xml_list

### ISSUE THREAD

**Issue by giansalex** [goetas-webservices/xsd2php#118]
Hello, I don't know if this is an error, i try `xsd2php` with [UBL 2.1 xsd](https://docs.oasis-open.org/ubl/os-UBL-2.1/xsd/maindoc/UBL-Invoice-2.1.xsd)

This is autogenerated yaml for InvoiceLine section
_InvoiceType.yaml_
```yaml
        invoiceLine:
            expose: true
            access_type: public_method
            serialized_name: InvoiceLine
            xml_element:
                cdata: false
                namespace: 'urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2'
            accessor:
                getter: getInvoiceLine
                setter: setInvoiceLine
            xml_list:
                inline: true
                entry_name: InvoiceLine
                namespace: 'urn:oasis:names:specification:ubl:schema:xsd:Invoice-2'
            type: array<TestNs\Ubl\CommonAggregateComponents\InvoiceLine>
```

After serialization, i get:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2">
  <cbc:UBLVersionID>2.1</cbc:UBLVersionID>
  <cbc:CustomizationID>2.0</cbc:CustomizationID>
  <cbc:ID>F001-2121</cbc:ID>
  <InvoiceLine>
    <cbc:ID>1</cbc:ID>
    <cbc:InvoicedQuantity>12.0</cbc:InvoicedQuantity>
  </InvoiceLine>
  <InvoiceLine>
    <cbc:ID>2</cbc:ID>
    <cbc:InvoicedQuantity>7.342</cbc:InvoicedQuantity>
  </InvoiceLine>
</Invoice>
```
> `<InvoiceLine>` don't have prefix `cac`

Then I change `xml_list:namespace` same to `xml_element:namespace`
```yaml
            xml_list:
                inline: true
                entry_name: InvoiceLine
                namespace: 'urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2'
```

After, i get expected result:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2">
  <cbc:UBLVersionID>2.1</cbc:UBLVersionID>
  <cbc:CustomizationID>2.0</cbc:CustomizationID>
  <cbc:ID>F001-2121</cbc:ID>
  <cac:InvoiceLine>
    <cbc:ID>1</cbc:ID>
    <cbc:InvoicedQuantity>12.0</cbc:InvoicedQuantity>
  </cac:InvoiceLine>
  <cac:InvoiceLine>
    <cbc:ID>2</cbc:ID>
    <cbc:InvoicedQuantity>7.342</cbc:InvoicedQuantity>
  </cac:InvoiceLine>
</Invoice>
```

**Comment by goetas** [comment]
This might be a bug in this library.

The `InvoiceLine` element in `urn:oasis:names:specification:ubl:schema:xsd:Invoice-2` is imported using `ref` from `urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2`, probably it should stay in the `urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2` namespace.

**Comment by anto-acube** [comment]
Hello, I'm experiencing the same issue, I think that this PR is related https://github.com/goetas-webservices/xsd2php/pull/108

**Comment by goetas** [comment]
fixed in 0.4.0
